src/main/java/org/jitsi/webrtcvadwrapper/audio/ByteSignedPcmAudioSegment.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright @ 2019-Present 8x8, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.webrtcvadwrapper.audio;
+
+import java.util.*;
+
+/**
+ * An audio segment which is stored as 16 bit signed PCM audio, but represented
+ * in byte pairs, where the first byte carries the least significant bytes.
+ *
+ * @author Nik Vaessen
+ */
+public class ByteSignedPcmAudioSegment
```

Comment:
Is this class used anywhere? Will it be used later?

Reply:
it used in the `SilenceFilter` in Jigasi: see https://github.com/jitsi/jigasi/pull/144/files#diff-4634c34c5271000480640320ae92b8de